### PR TITLE
operatorhub-workflow

### PR DESCRIPTION
**Pull request by DanielePalaia**
Adding a workflow to publish an OLM package on OperatorHub and Openshift Ecosystem

### CODE REVIEW

File: .github/workflows/operatorhub.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          TOKEN: ${{ secrets.OPERATORHUB_TOKEN }}
+        run: |
+          git clone https://github.com/rabbitmq/community-operators
+          cd community-operators/operators/rabbitmq-single-active-consumer-operator
+          git branch rabbitmq-single-active-consumer-${{ env.RELEASE_VERSION }}
+          git checkout rabbitmq-single-active-consumer-${{ env.RELEASE_VERSION }}
+          cp -fR /home/runner/work/single-active-consumer-operator/single-active-consumer-operator/OLM-Package-Repo/OLM2/rabbitmq-single-active-consumer-operator/${{ env.RELEASE_VERSION }} .
+          git add .
+          git commit -s -m "RabbitMQ single active consumer operator new release"
+          git push https://DanielePalaia:"$TOKEN"@github.com/rabbitmq/community-operators
```
Comment (ablease) [inline .github/workflows/operatorhub.yml:58]: Could we use a seperate team account for this? We probably have some thing in lastpass we could use.

File: .github/workflows/operatorhub.yml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          TOKEN: ${{ secrets.OPERATORHUB_TOKEN }}
+        run: |
+          git clone https://github.com/rabbitmq/community-operators-prod
+          cd community-operators-prod/operators/rabbitmq-single-active-consumer-operator
+          git branch rabbitmq-single-active-consumer-${{ env.RELEASE_VERSION }}
+          git checkout rabbitmq-single-active-consumer-${{ env.RELEASE_VERSION }}
+          cp -fR /home/runner/work/single-active-consumer-operator/single-active-consumer-operator/OLM-Package-Repo/OLM2/rabbitmq-single-active-consumer-operator-openshift/${{ env.RELEASE_VERSION }} .
+          git add .
+          git commit -s -m "RabbitMQ single active consumer new release"
+          git push https://DanielePalaia:"$TOKEN"@github.com/rabbitmq/community-operators-prod
```
Comment (ablease) [inline .github/workflows/operatorhub.yml:72]: Same as above re team account.

File: .github/workflows/operatorhub.yml (hunk 1 above)
Comment (DanielePalaia) [reply .github/workflows/operatorhub.yml:58]: @ablease Not really, the fact is that to publish to an open source project an account needs to be authorized by the VMware opensource project. I tried to see with them if a bot user can authorized and it doesn't seem so, so I'm afraid this is the only way...

Comment (ablease) [reply .github/workflows/operatorhub.yml:58]: Ah ok. Hopefully we can change it in the future. :)

**Review by ablease — APPROVED** [review]

**Comment by DanielePalaia** [comment]
@ablease Thank you!

**Pull request merged**